src/app/App.tsx

```diff
@@ -114,7 +124,14 @@ const App: React.ComponentType = () => {
                     <Routes />
                 </>
             ) : (
-                <NotAuthorized serviceName={ serviceName } />
+                <>
+                    <NotAuthorized
+                        description={ `Contact your organization administrator for more information or visit
+                        ${ myUserAccess } to learn more about your permissions. To manage your notifications,
+                        go to your ${ userPreferences }.` }
```

Comment:
Something similar to:

```suggestion
                        description={ <> 
                        Contact your organization administrator for more information or visit
                        { myUserAccess } to learn more about your permissions. To manage your notifications,
                        go to your { userPreferences }.
                        </> }
```